enforcer-parent/enforcer/src/main/java/org/wso2/choreo/connect/enforcer/api/RestAPI.java

```diff
@@ -146,6 +147,9 @@ public String init(Api api) {
                             api.getEndpointSecurity().getSandBoxSecurityInfo()));
         }
 
+        org.wso2.choreo.connect.enforcer.commons.model.ChoreoComponentInfo choreoComponentInfo = getChoreoComponentInfo(
```

Comment:
Shall we import the class

Reply:
a class with the same name already imported. Hence had to improve with the fully qualified class name.